web/src/app/util/Options.js

```diff
@@ -16,7 +16,6 @@ import Dialog from '@material-ui/core/Dialog'
 import DialogTitleWrapper from '../dialogs/components/DialogTitleWrapper'
```

Comment:
Can we delete this file, in favor of the newer `OtherActions`?

Reply:
I started to, but it's used in the alerts stuff where the code depends on the mutation behaviors so I decided that'd be better in it's own PR.